### ISSUE TITLE
Ask for test

### ISSUE THREAD

**Issue by zengdonghuo** [PKU-ICST-MIPL/UGACH_AAAI2018#7]
Hi

Thanks for releasing your dataset, training set but how about the rest test set? if I get the training set without a test set. I think I have no way to go. can you share this part?

Best Regards
DH

**Comment by PKU-ICST-MIPL** [comment]
you can download the nus-wide dataset, and use the feature extract tools to extract the feature for test set. We resized the images to 255*225 and crop into 224*224(crop is achieved by Caffe). the query set is randomly selected from the test set.

**Comment by zengdonghuo** [comment]
thanks for your reply, the problem is that many images are offline, I can't download.

**Comment by fushuiwu** [comment]
**Hello, does the test set need to be sampled from the NUS-WIDE raw dataset and extracted through feature extraction?**
